### ISSUE TITLE
getters/settings not showing on docs website

### ISSUE THREAD

**Issue by Ehesp** [invertase/react-native-firebase#3844]
## Documentation Feedback

E.g. https://rnfirebase.io/reference/auth - `languageCode` not showing.

---

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by russellwheatley** [comment]
Hmmm, the problem is the [getters & setters](https://github.com/invertase/react-native-firebase/blob/master/packages/auth/lib/index.d.ts#L1177) appear to be ignored for the `typeDoc.json` reference (I guess because they're not technically properties). I had to delete and add it as a standard property for it to be included into the generated `typeDoc.json`. I changed the auth `index.d.ts` to below screenshot:
![Screenshot 2020-07-09 at 09 44 30](https://user-images.githubusercontent.com/16018629/87018620-68db2100-c1c9-11ea-806d-8bcad1998b62.png)

Not sure if it's acceptable. Let me know and I'll update accordingly
![Screenshot 2020-07-09 at 09 46 14](https://user-images.githubusercontent.com/16018629/87018778-a049cd80-c1c9-11ea-9f3c-46756fc5501b.png)
